bazel/repositories.bzl

```diff
@@ -42,7 +42,7 @@ def proxy_wasm_cpp_host_repositories():
     maybe(
         http_archive,
         name = "bazel-zig-cc",
-        sha256 = "ad6384b4d16ebb3e5047df6548a195e598346da84e5f320250beb9198705ac81",
+        sha256 = "b3dd78971f0f831f62d84ae96656bdb0941b91b7c80097d69f5be213351cc6ff",
         strip_prefix = "bazel-zig-cc-v0.4.4",
         url = "https://git.sr.ht/~motiejus/bazel-zig-cc/archive/v0.4.4.tar.gz",
```

Comment:
having said that, could you change this URL to https://github.com/uber/bazel-zig-cc per https://git.sr.ht/~motiejus/bazel-zig-cc ?